exchange-client.gemspec

```diff
@@ -0,0 +1,30 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'openstax/exchange/client/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "openstax-exchange-client"
```

Comment:
Ok.  I changed the code to be more consistent.

The `openstax/exchange-ruby` repo now holds the `openstax_exchange` gem which uses OpenStax::Exchange as its primary namespace.  Now you can say things like:
```ruby
  OpenStax::Exchange.configure do |config|
    # ...
  end

  OpenStax::Exchange.use_real_client

  OpenStax::Exchange.create_identifier
```
which is what I think you were wanting.